client/src/components/DetailView.tsx

```diff
@@ -207,15 +212,22 @@ class DetailViewWithoutI18n extends Component<Props, State> {
                         {Helpers.titleCase(detailAddr.streetname)},{" "}
                         {Helpers.titleCase(detailAddr.boro)}
                       </h4>
-                      {!Helpers.addrsAreEqual(detailAddr, searchAddr) && (
-                        <a // eslint-disable-line jsx-a11y/anchor-is-valid
-                          onClick={() => this.setState({ showCompareModal: true })}
-                        >
-                          <Trans render="i">
-                            How is this building associated to this portfolio?
-                          </Trans>
-                        </a>
-                      )}
+                      {!Helpers.addrsAreEqual(detailAddr, searchAddr) &&
+                        (useNewPortfolioMethod ? (
+                          <Link to={this.props.addressPageRoutes.summary}>
+                            <i>
+                              <BldgAssociationHeader />
+                            </i>
+                          </Link>
+                        ) : (
+                          <a // eslint-disable-line jsx-a11y/anchor-is-valid
+                            onClick={() => this.setState({ showCompareModal: true })}
+                          >
+                            <i>
+                              <BldgAssociationHeader />
```

Comment:
This sounds like a tenant association within a building! lol i got confused. Maybe PortfolioAssociationHeader?

Reply:
Ok did a quick profile and it seems like the computation is pretty instant even for our largest portfolio! Check out this cute pic too and see if u can spot our new bbls: 
![image](https://user-images.githubusercontent.com/12834575/135516654-799ee235-277b-44c2-9dc8-894aefa53ae1.png)